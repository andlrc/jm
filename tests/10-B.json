{
    "@extends": ["10-A.json"],
	"sequence": [
		{
			"@insert": 1,
			"@value": "insertedField"
		}
	]
}
