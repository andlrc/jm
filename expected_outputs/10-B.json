{
	"sequence": [
		"fieldA",
		"insertedField",
		"fieldB",
		"fieldC"
	]
}
